src/common/_tempMDL2Icons.scss

```diff
@@ -0,0 +1,16 @@
tlU0AuHAAIK7cMWEg4KBkACCu3DUM3Kx8TAAgrtw4sJBwUDAAIK7cPHhgTDggACCu3EA4MCQcEAAgrtxEREREREQAIK7cSsJBwUDAACCsAtwGFbVU9JQAIK7cCZVNALhwACCu3A1hIOCgZAAgrtwRDNysfEwAIK7cFLCQcFAwACCu3Bh4ZEw4IAAgrtwcODAkHBAAIK7cIEREREREACCu3CbCQcFAwAAgrALITAQcrsAAgRX1pGEQAACoAVQBwAIAAqgEAAYADQAHAAEAAVQBwAIAAqgEAAYADQAHAAEAAAAAAAAAAAAAUABQAFAAUAGkAAgCmAAAEhAV2AAMABwAAMxEhESUhESGmA978tgK2/UoFdvqKlAROAAMAAACACAAHAAARACQAMwAAAR4DFRQOAiMiJicJAiEJAREuBTU0NjMyFhc1IQkBMj4CNTQuAicRAR4BBoBYjmQ2ZK7phWjFU/8A/QADgAMA/IADABI0OTgtHCYaMV8w/bX9AATLarqLUSRCXzv93ECWBcsneZmyYIXprmRBP/8AAwADgPo1AwABOwQDAwYPGhcaJgILjf0A/gBRi7pqRIJzYCL+xf3cLS8AAAAAACUBwgADAAEEAwACAAwAAAADAAEEBQACABAADAADAAEEBgACAAwAHAADAAEEBwACABAAKAADAAEECAACABAAOAADAAEECQAAAGQASAADAAEECQABABgArAADAAEECQACAA4AxAADAAEECQADABgArAADAAEECQAEABgArAADAAEECQAFABgA0gADAAEECQAGABgArAADAAEECQAHAHIA6gADAAEECQAIACoBXAADAAEECQALAFQBhgADAAEECQANApgB2gADAAEECQAOAFQBhgADAAEECgACAAwAAAADAAEECwACABAEcgADAAEEDAACAAwAAAADAAEEDgACAAwEggADAAEEEAACAA4EjgADAAEEEwACABIEnAADAAEEFAACAAwAAAADAAEEFQACABAErgADAAEEFgACAAwAAAADAAEEGQACAA4EvgADAAEEGwACABAEzAADAAEEHQACAAwAAAADAAEEHwACAAwAAAADAAEEJAACAA4E3AADAAEEKgACAA4E6gADAAEELQACAA4E+AADAAEICgACAAwAAAADAAEIFgACAAwAAAADAAEMCgACAAwAAAADAAEMDAACAAwAAABOAG8AcgBtAGEAbABvAGIAeQENAGUAagBuAOkAbgBvAHIAbQBhAGwAUwB0AGEAbgBkAGEAcgBkA5oDsQO9A78DvQO5A7oDrACpACAAMgAwADEANAAgAE0AaQBjAHIAbwBzAG8AZgB0ACAAQwBvAHIAcABvAHIAYQB0AGkAbwBuAC4AIABBAGwAbAAgAFIAaQBnAGgAdABzACAAUgBlAHMAZQByAHYAZQBkAC4ARgB1AGwAbABNAEQATAAyAC4AMQAuADgAUgBlAGcAdQBsAGEAcgBWAGUAcgBzAGkAbwBuACAAMwAuADAAMABTAGUAZwBvAGUAIABpAHMAIABhACAAdAByAGEAZABlAG0AYQByAGsAIABvAGYAIAB0AGgAZQAgAE0AaQBjAHIAbwBzAG8AZgB0ACAAZwByAG8AdQBwACAAbwBmACAAYwBvAG0AcABhAG4AaQBlAHMALgBNAGkAYwByAG8AcwBvAGYAdAAgAEMAbwByAHAAbwByAGEAdABpAG8AbgBoAHQAdABwADoALwAvAHcAdwB3AC4AbQBpAGMAcgBvAHMAbwBmAHQALgBjAG8AbQAvAHQAeQBwAG8AZwByAGEAcABoAHkALwBmAG8AbgB0AHMALwBZAG8AdQAgAG0AYQB5ACAAdQBzAGUAIAB0AGgAaQBzACAAZgBvAG4AdAAgAGEAcwAgAHAAZQByAG0AaQB0AHQAZQBkACAAYgB5ACAAdABoAGUAIABFAFUATABBACAAZgBvAHIAIAB0AGgAZQAgAHAAcgBvAGQAdQBjAHQAIABpAG4AIAB3AGgAaQBjAGgAIAB0AGgAaQBzACAAZgBvAG4AdAAgAGkAcwAgAGkAbgBjAGwAdQBkAGUAZAAgAHQAbwAgAGQAaQBzAHAAbABhAHkAIABhAG4AZAAgAHAAcgBpAG4AdAAgAGMAbwBuAHQAZQBuAHQALgAgAFkAbwB1ACAAbQBhAHkAIABvAG4AbAB5ACAAKABpACkAIABlAG0AYgBlAGQAIAB0AGgAaQBzACAAZgBvAG4AdAAgAGkAbgAgAGMAbwBuAHQAZQBuAHQAIABhAHMAIABwAGUAcgBtAGkAdAB0AGUAZAAgAGIAeQAgAHQAaABlACAAZQBtAGIAZQBkAGQAaQBuAGcAIAByAGUAcwB0AHIAaQBjAHQAaQBvAG4AcwAgAGkAbgBjAGwAdQBkAGUAZAAgAGkAbgAgAHQAaABpAHMAIABmAG8AbgB0ADsAIABhAG4AZAAgACgAaQBpACkAIAB0AGUAbQBwAG8AcgBhAHIAaQBsAHkAIABkAG8AdwBuAGwAbwBhAGQAIAB0AGgAaQBzACAAZgBvAG4AdAAgAHQAbwAgAGEAIABwAHIAaQBuAHQAZQByACAAbwByACAAbwB0AGgAZQByACAAbwB1AHQAcAB1AHQAIABkAGUAdgBpAGMAZQAgAHQAbwAgAGgAZQBsAHAAIABwAHIAaQBuAHQAIABjAG8AbgB0AGUAbgB0AC4ATgBvAHIAbQBhAGEAbABpAE4AbwByAG0A4QBsAE4AbwByAG0AYQBsAGUAUwB0AGEAbgBkAGEAYQByAGQATgBvAHIAbQBhAGwAbgB5BB4EMQRLBEcEPQRLBDkATgBvAHIAbQDhAGwAbgBlAE4AYQB2AGEAZABuAG8AdABoAbABoQMAAG4AZwBBAHIAcgB1AG4AdABhAAMAAAAAAAD/TgB3AAAAAAAAAAAAAAAAAAAAAAAAAAAAAQACAAgACv//AA8=') format('truetype');
```

Comment:
If you just needed 1 icon, couldn't you simply embed the font containing a single icon here?

It seems like you're embedding ALL of the icons here. This really isn't the right place to do this either, it should be done in products. Doing it here means everyone will merge this into their javascript, including products that have their own icon fonts.